UnityProject/Assets/Scripts/Systems/Score/RoundEndScoreBuilder.cs

```diff
@@ -0,0 +1,102 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Managers;
+using NSubstitute.ReceivedExtensions;
+using Shared.Managers;
+using UnityEngine;
+
+namespace Systems.Score
+{
+	public class RoundEndScoreBuilder : SingletonManager<RoundEndScoreBuilder>
+	{
+		/// <summary>
+		/// How much does score entry that returns true or false score?
+		/// </summary>
+		[SerializeField] private int boolScore = 10;
+		[SerializeField] private int negativeModifer = 5;
+		[SerializeField] private Occupation captainOccupation;
+
+		public override void Start()
+		{
+			base.Start();
+			EventManager.AddHandler(Event.RoundEnded, CalculateScoresAndShow);
+			EventManager.AddHandler(Event.RoundStarted, CreateCommonScoreEntries);
+		}
+
+		private void CreateCommonScoreEntries()
+		{
+			ScoreMachine.AddNewScoreEntry("laborPoints", "Total Labor Points", ScoreMachine.ScoreType.Int, ScoreCategory.StationScore, ScoreAlignment.Good);
+			ScoreMachine.AddNewScoreEntry("randomEventsTriggered", "Random Events Endured", ScoreMachine.ScoreType.Int, ScoreCategory.StationScore, ScoreAlignment.Good);
+			ScoreMachine.AddNewScoreEntry("foodmade", "Meals Prepared", ScoreMachine.ScoreType.Int, ScoreCategory.StationScore, ScoreAlignment.Good);
+			ScoreMachine.AddNewScoreEntry("hostileNPCdead", "Hostile NPCs dead", ScoreMachine.ScoreType.Int, ScoreCategory.StationScore, ScoreAlignment.Good);
+			ScoreMachine.AddNewScoreEntry("healing", "Healing Done", ScoreMachine.ScoreType.Int, ScoreCategory.StationScore, ScoreAlignment.Good);
+			ScoreMachine.AddNewScoreEntry("foodeaten", "Food Eaten", ScoreMachine.ScoreType.Int, ScoreCategory.StationScore, ScoreAlignment.Weird);
+			ScoreMachine.AddNewScoreEntry("explosions", "Number of explosions this round", ScoreMachine.ScoreType.Int, ScoreCategory.StationScore, ScoreAlignment.Weird);
+			ScoreMachine.AddNewScoreEntry("clownBeaten", "Clown Abused", ScoreMachine.ScoreType.Int, ScoreCategory.StationScore, ScoreAlignment.Bad);
```

Comment:
Some of these are defined as constants in other files. Could you define the constants in this class and access them from the other scripts?

Reply:
done